# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375531720 -7200
#      Sat Aug 03 14:08:40 2013 +0200
# Node ID f29de5a997a759e15c052df98c7d74f770e3ee4a
# Parent  04da75219e19a94dddfa63c038cdb2c84ec96d8e
-Fix (r19328): Right padding of the airport name string should be negative.

diff --git a/src/airport_gui.cpp b/src/airport_gui.cpp
--- a/src/airport_gui.cpp
+++ b/src/airport_gui.cpp
@@ -335,7 +335,7 @@ public:
 					if (!as->IsAvailable()) {
 						GfxFillRect(r.left + 1, y + 1, r.right - 1, y + this->line_height - 2, PC_BLACK, FILLRECT_CHECKER);
 					}
-					DrawString(r.left + WD_MATRIX_LEFT, r.right + WD_MATRIX_RIGHT, y + WD_MATRIX_TOP, as->name, ((int)i == _selected_airport_index) ? TC_WHITE : TC_BLACK);
+					DrawString(r.left + WD_MATRIX_LEFT, r.right - WD_MATRIX_RIGHT, y + WD_MATRIX_TOP, as->name, ((int)i == _selected_airport_index) ? TC_WHITE : TC_BLACK);
 					y += this->line_height;
 				}
 				break;
